### PR TITLE
feat: style package for theming and styled components

### PR DESCRIPTION
**Pull request by suddjian**
💔 Breaking Changes

🏆 Enhancements

Adds a new package. For now the package contents are just porting the theme from Superset. This is so it can be used by viz plugins and other superset-ui components.

📜 Documentation

See readme

🐛 Bug Fix

🏠 Internal

### CODE REVIEW

File: packages/superset-ui-style/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "style",
```
Comment (kristw) [inline packages/superset-ui-style/package.json:2]:
```ts
"name": "@superset-ui/style"
```

File: packages/superset-ui-style/yarn.lock [hunk 2]
```diff
@@ -0,0 +1,368 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (kristw) [inline packages/superset-ui-style/yarn.lock:1]: please remove this file. we only check in the `yarn.lock` at root level

File: packages/superset-ui-style/package.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/apache-superset/superset-ui.git"
+  },
+  "keywords": [
+    "superset"
+  ],
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
```
Comment (kristw) [inline packages/superset-ui-style/package.json:22]: please add simple unit test to maintain 100% test coverage for core package.

File: packages/superset-ui-style/package.json (hunk 1 above)
Comment (suddjian) [reply packages/superset-ui-style/package.json:2]: oops, thanks for catching that 😐

File: packages/superset-ui-style/package.json (hunk 3 above)
Comment (kristw) [reply packages/superset-ui-style/package.json:22]: can remove this script

File: packages/superset-ui-style/README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# `style`
```
Comment (kristw) [inline packages/superset-ui-style/README.md:1]: `README` for the other packages have `npm` and `david` badges on top. Would be nice to keep the same format.

File: packages/superset-ui-style/package.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@superset-ui/style",
+  "version": "0.12.13",
+  "description": "Component styles for Superset",
+  "author": "Aaron Suddjian <aaron@preset.io>",
+  "homepage": "https://github.com/apache-superset/superset-ui#readme",
+  "license": "Apache-2.0",
+  "main": "lib/index.js",
```
Comment (kristw) [inline packages/superset-ui-style/package.json:8]: Add

```
  "sideEffects": false,
```

for more effective tree-shaking

File: packages/superset-ui-style/README.md [hunk 6]
```diff
@@ -1,5 +1,8 @@
 # `style`
```
Comment (kristw) [inline packages/superset-ui-style/README.md:1]: nit: @superset-ui/style

**Pull request merged**